driver/src/main/java/org/neo4j/driver/Config.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j.driver;
+
+import java.util.logging.Level;
+
+import org.neo4j.driver.internal.logging.JULogging;
+import org.neo4j.driver.internal.spi.Logging;
+
+/**
+ * A configuration class to config driver properties.
+ * <p>
+ * To create a config:
+ * <pre>
+ * {@code
+ * Config config = Config
+ *                  .builder()
+ *               ** .setProperties() **
+ *                  .build();
+ * }
+ * </pre>
+ */
+public class Config
+{
+
+    /** A config with all default settings */
+    public static final Config DEFAULT_CONFIG = Config.builder().build();
```

Comment:
`.builder().build()` is kind of awkward. Might want to think about the naming there.